src/Gateway.php

```diff
@@ -358,6 +360,39 @@ public function tokenize($card_holder, $card_number, $expiry_date, $cvv) {
         return $this->do_request('POST', '/credit_cards', $payload);
     }
 
+    /**
+     * Fetch the details of a previously tokenized credit card
+     * @param string $token the card token
+     * @return \StdObject
+     */
+    public function get_tokenized_card($token) {
+        if(is_null($token) || (strlen($token) === 0)) throw new InvalidArgumentException('Token is a required field.');
+        return $this->do_request('GET', '/credit_cards/'.$token);
+    }
+
+	/**
+	 * Update a tokenized card
+	 * @param string $token the card token
+	 * @param string $expiry_date the card expiry date (mm/yyyy format)
+	 * @param string $alias alias to associate with the card
+	 * @return \StdObject
+	 */
+    public function update_tokenized_card($token, $expiry_date=null, $alias=null) {
+		if(is_null($token) || (strlen($token) === 0)) throw new InvalidArgumentException('Token is a required field.');
+
+		$payload = null;
```

Comment:
this should be `[]` so the index operations in the following if statements do not fail their index assignment. If the api/client code requires null it should detect an empty array and switch inside do_request so the api logic is contained there